hack/tests/e2e-ansible-molecule.sh

```diff
@@ -1,69 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
@fabianvf since you are modifying this file for the ansible-plugin, I assume @varshaprasad96 should leave this file untouched?

Reply:
correct

Reply:
they've been rewritten in the scaffold PR, this will just cause a conflict

Reply:
Ill add this, and comment the targets in Makefile where its called.

Reply:
e2e tests will fail if the scaffolding is removed. so we either need to block this PR on the plugin PR

Or have the plugin PR rebase and re-add these tests.

Either way is fine by me.

Reply:
I understand that the problem here is that this tests were not port yet.

Reply:
I create a task for that: https://github.com/operator-framework/operator-sdk/issues/3540